doc/bt/BC_NORMAL_SetPosition.bt

```diff
@@ -0,0 +1,23 @@
+//------------------------------------------------
```

Comment:
Shouldn't this be part of BC_NORMAL? I assume you wanted to split it off for cleanliness, but it will only be useful as a template if you can apply it to any BC_NORMAL and get the correct result. With it split off you would have to know what the specific BC_NORMAL does to look up the template.

If we want it cleaner (I don't know how big BC_NORMAL is), maybe we could split off only the subop specific variables, and `#include` the files in the parent template.

Reply:
In the case of `BC_NORMAL` it is around 20. For the `ZC_NORMAL` packets, there might be a few thousand.

I think splitting off the parent template and children would work fine.

Reply:
Yea, though on the other hand, you're saying thousands, do we want to litter the folder with thousands of sub-template files?^^"

Reply:
Not really, I was assuming a two child folders - one for `BC_NORMAL` and the other for `ZC_NORMAL`.

Reply:
Yea, still a *lot* of files to track and clone. But oh well. I think we should see how it goes. One giant ZC_NORMAL template might not be that terrible either, that way you can see everything at once.